### DISCUSSION TITLE
20241103_advanced 版的 "advanced" 指的是有哪些新功能？

### DISCUSSION THREAD

**Discussion by sjackwu** [oniondelta/Onion_Rime_Files#34] [Q&A] [answered]
最近的版本 20241103_advanced，裏面有個關於 advanced 的說明，照做之後也沒看到什麼新的功能...

請問，這個新功能有什麼特別的地方嗎？ 😁

**Comment by oniondelta** [comment, marked as answer]
沒新功能！😅
特此標注是改寫用到[ librime 新的功能設定](https://github.com/rime/librime/pull/691)！
於 20241123 當前官網封裝的〔鼠鬚管〕和〔小狼毫〕，無法使用該功能設定！（掛接方案會失效！）
須置換 librime 核心才能正常作用！
只對注音 mix-in 和注音 plus 方案有改寫和影響！

該新功能設定可免除於注音 mix-in 和 plus 方案輸入「'」「/」「數字」時，數字太長會跳回掛接方案的 Bug！
之前用正則限定，有長度限制！新的 librime 設定可避免！
承上，因此打字時，程式可減少判斷複雜的正則，效能可能較好？！
